### ISSUE TITLE
JAWS mislabels combobox as 'mnu' and fails to announce listbox item positions when nested in menubar on Chrome or Edge

### ISSUE THREAD

**Issue by YunHanBai** [FreedomScientific/standards-support#831]
### Summary

When navigating a web page with JAWS screen reader, `combobox` elements inside a `menubar` are incorrectly labeled as 'mnu' in the Braille Viewer instead of 'cbo'. Additionally, JAWS does not announce the position of items in a `listbox` if the listbox is inside a `menubar`, despite having `role="listbox"` on the container and `role="option"` on the child elements.

`combobox` and `listbox` function correctly if they are not inside a `menubar`.

This issue occurs in Chrome/Edge but not in Firefox.

### Steps to reproduce:

1. Open Chrome or Firefox with JAWS screen reader enabled.
2. Navigate to the provided CodeSandbox example: https://codesandbox.io/p/sandbox/comboboxa11y-simplified-hyt9h6
3. Focus on the `combobox` inside the menubar and observe the labeling in the Braille Viewer.
4. Navigate the items within the listbox inside the menubar and listen for the announcement of item position and total count.

### Expected result
Similar to `combobox` and `listbox` when placed outside a `menubar`, the following behaviors are expected:
- The `combobox` should be labeled as 'cbo' in the Braille Viewer.
- The screen reader should announce the position of the items and the total count within the `listbox`.

### Actual result

In Chrome/Edge, the `combobox` is labeled as 'mnu' in the Braille Viewer, and the screen reader does not announce the position and total count of items within the `listbox`. In Firefox, the `combobox` is correctly labeled as 'cbo', and the position and total count are announced.

### Example

The issue can be replicated using the following CodeSandbox example:
https://codesandbox.io/p/sandbox/comboboxa11y-simplified-hyt9h6

#### JAWS version and build number
JAWS 2024.2403.3 (also tested with JAWS 2022.2211.7)
 
#### Operating System and version
Windows 11 Enterprise 23H2

#### Browser and version:
Chrome: 124.0.6367.119
Edge: 124.0.2478.80
Firefox: 125.0.3

**Comment by JAWS-test** [comment]
A menubar must not contain a comobox or listbox, so there is no JAWS error here, but an error in your example.

See https://www.w3.org/TR/wai-aria-1.3/#menubar

**Comment by JAWS-test** [comment]
If you want listbox and combobox in an area that can be operated with arrow keys, please use role=[toolbar ](https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/)instead of menubar

**Comment by YunHanBai** [comment]
Hello @JAWS-test,

Thank you for the timely response.

I've been reviewing the WAI-ARIA 1.3 documentation you mentioned, it doesn't seem to explicitly state that a `combobox` or `listbox` cannot be included within a `menubar` or `menuitem`.

Would you be able to point me towards any specific documentation that clearly outlines that a menubar must not contain controls like `combobox` or `listbox`? Or could you confirm whether JAWS will definitely not support `combobox` or `listbox` when they are in a menubar?

I'd really appreciate your help in clarifying this!

**Comment by JAWS-test** [comment]
> Or could you confirm whether JAWS will definitely not support combobox or listbox when they are in a menubar?

No, because I am not from Freedom Scientific

> Would you be able to point me towards any specific documentation that clearly outlines that a menubar must not contain controls like combobox or listbox?

Yes. See "Allowed Accessibility Child Roles"

**Comment by stevefaulkner** [comment]
> Or could you confirm whether JAWS will definitely not support combobox or listbox when they are in a menubar?

JAWS endeavors to support web standards. As `combobox` or `listbox` is not an allowed child of `menubar` it is very unlikely that it will be supported by JAWS. /cc @BrettLewisVispero

**Comment by BrettLewisVispero** [comment]
Agree with @stevefaulkner. As the ARIA standard does not allow this as a child role, we will not be making an effort to support these types of controls in a menubar.
